lc-hax/Scripts/Patches/UnlockableSuitPatch.cs

```diff
@@ -0,0 +1,41 @@
+using HarmonyLib;
+using Hax;
+
+[HarmonyPatch(typeof(UnlockableSuit))]
+public static class UnlockableSuitPatch {
+
+    private static bool overrideSuit = false;
+    private static int customSuitID = -1;
+    private static int originalSuitID = -1;
+
+    public static void SetPlayerSuit(Unlockable suitID) {
+        if (Helper.LocalPlayer is null) return;
+        customSuitID = (int)suitID;
+        originalSuitID = -1;
+        overrideSuit = true;
+        UnlockableSuit[] unlockableSuits = UnityEngine.Object.FindObjectsOfType<UnlockableSuit>();
+        foreach (UnlockableSuit suit in unlockableSuits) {
+            if (suit.NetworkManager.IsListening) {
+                suit.SwitchSuitToThis(Helper.LocalPlayer);
+                break;
+            }
+        }
+    }
```

Comment:
Move this into the suit command script.

Reply:
it sets the variables, but try to improve it as much you want

Reply:
Oh right, I didn't notice that.